pkg/user/tx_client_test.go

```diff
@@ -594,7 +604,7 @@ func setupTxClient(
 }
 
 func setupTxClientWithDefaultParams(t *testing.T, opts ...user.Option) (encoding.Config, *user.TxClient, testnode.Context) {
-	return setupTxClient(t, 0, 8388608, opts...) // no ttl and 8MiB block size
+	return setupTxClient(t, 0, 128, 8388608, opts...) // no ttl and 8MiB block size
```

Comment:
if we want or need a default config for tests here then imo we should just have a function that returns that and not have this function